packages/commonjs/test/fixtures/form/defaultIsModuleExports-false-no-default/output.js

```diff
@@ -0,0 +1,9 @@
+var named = 3;
+
+var input = {
+	named: named
+};
+
+export default input.default;
```

Comment:
I'm not sure about this one. Should we avoid injecting the default export when it's not present? But then, how do we handle cases where `input.default` exists but it's not easily statically analyzable?

Reply:
Good question. I would lean towards adding it always for now, though maybe as `undefined`, as this is similar to the current behaviour.